### PR TITLE
Conversation API: Adding Webhook Management page 

### PR DESCRIPTION
**Pull request by rumen-kyusakov**

### CODE REVIEW

File: docs/conversation/conversation-callbacks.md [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+  Registering webhooks and listening to callbacks from Sinch Conversation API.
+hidden: false
+---
+
+## Webhook Management <span class="betabadge">Beta</span>
+
+Conversation API delivers contact messages, delivery receipts for app messages and various notifications through callbacks.
+API clients can create fine-grained subscriptions for up-to 5 endpoints (webhooks) per Conversation API app
+through the [Sinch Portal](https://dashboard.sinch.com/convapi/apps) or using **/webhooks** management endpoint.
+Each webhook defines a subscription for events defined by a list of triggers. The events are delivered by Conversation API
```
Comment (matokeni) [inline docs/conversation/conversation-callbacks.md:13]: too much "define", maybe:
Each webhook represents a subscription blabla..

File: docs/conversation/conversation-callbacks.md [hunk 2]
```diff
@@ -0,0 +1,739 @@
...
+| longitude     | float              | Longitude of the geographic coordinate. |
+
+##### Choice Response Message
+
+The choice response message represents a contact response to a choice message.
+
+| Field         | Type               | Description                                       |
+| ------------- | ------------------ | ------------------------------------------------- |
+| message_id    | string             | The message id containing the choice.             |
+| postback_data | string             | The postback data defined in the selected choice. |
```
Comment (matokeni) [inline docs/conversation/conversation-callbacks.md:178]: maybe we can mention that if it was not defined, we have a default postback

File: docs/conversation/conversation-callbacks.md [hunk 3]
```diff
@@ -0,0 +1,739 @@
...
+- `FAILED`
+- `SWITCHING_CHANNEL`
+
+Each message and event sent by the API clients to contacts go through the following states:
+
+1. The message is successfully queued in Conversation API. It has status `QUEUED`.
+2. The message is successfully dispatched to an underlying channel. It has status `QUEUED_ON_CHANNEL`.
+3. A delivery report is sent from the underlying channel detailing the delivery status on the channel. Depending on the channel response and the processing state of the message the message is transitioned to one of following states:
+
+    3.1 `DELIVERED` - the channel delivery report indicated the message has reached the end user. Some channels can later sent new delivery report with status `READ`.
```
Comment (matokeni) [inline docs/conversation/conversation-callbacks.md:335]: sent -> send

File: docs/conversation/conversation-callbacks.md [hunk 4]
```diff
@@ -0,0 +1,739 @@
...
+      "email": "",
+      "external_id": "",
+      "metadata": "",
+      "language": "UNSPECIFIED"
+    }
+  }
+}
+```
+
+### Capability Check
```
Comment (matokeni) [inline docs/conversation/conversation-callbacks.md:616]: we can link here somewhere the Capability page that we have, just referring to it, "read more at..."

same for the Opt-In / Out 😇

File: docs/conversation/conversation-callbacks.md (hunk 1 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-callbacks.md]: yes, much better. Fixed

File: docs/conversation/conversation-callbacks.md (hunk 2 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-callbacks.md]: good point! fixed

File: docs/conversation/conversation-callbacks.md (hunk 3 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-callbacks.md]: done

File: docs/conversation/conversation-callbacks.md (hunk 4 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-callbacks.md:616]: done

**Pull request merged**